pkg/apis/alert/alert_test.go

```diff
@@ -106,23 +106,26 @@ func (s *alertTestSuite) Test_OnSync_ResourcesCreated() {
 		ObjectMeta: metav1.ObjectMeta{Name: appName},
 		Spec:       radixv1.RadixRegistrationSpec{AdGroups: []string{"admin"}, ReaderAdGroups: []string{"reader"}},
 	}
-	s.radixClient.RadixV1().RadixRegistrations().Create(context.Background(), rr, metav1.CreateOptions{})
-	radixalert := &radixv1.RadixAlert{
+	if _, err := s.radixClient.RadixV1().RadixRegistrations().Create(context.Background(), rr, metav1.CreateOptions{}); err != nil {
+		s.Require().NoError(err)
+	}
```

Comment:
```suggestion
	err := s.radixClient.RadixV1().RadixRegistrations().Create(context.Background(), rr, metav1.CreateOptions{})
	s.Require().NoError(err)
```
No need to wrap this in `if` block.

Reply:
Fixed